blog/2023-11-20-second-revival-milestone-unveiled/index.md

```diff
@@ -0,0 +1,105 @@
+A first success was _@skaldarnar_ identifying leftovers of the VR integration removed in revival milestone 1.
+Removing them not only further decluttered our codebase but contributed to a more resource-efficient gameplay experience by halving the required rendering passes.
+
+Together, our community discussed the benefit of having documentation closer to our code.
+As a result, we integrated our Terasology engine wiki into our codebase, making it our new ["Knowledge Base"](https://terasology.org/Terasology/#/) provided using Docsify on GitHub Pages, allowing to co-locate code and associated documentation changes in commits and PRs.
+All links to our old wiki should now point to our ["Knowledge Base"](https://terasology.org/Terasology/#/). If you find any broken links, please report them on Discord or using a GitHub issue 💡
+
+Next to these bigger efforts, a lot of smaller discussions and activities happened across our GitHub repositories and Discord channels.
+_@BSA_, _@Niruandaleth_, and _@skaldarnar_ met once per week virtually to discuss milestone progress, agree on strategic questions, and collaborate on blockers and reviews.
+Over the course of our second revival milestone, _@Cervator_ provided guidance for new community members and revivers alike, sharing his knowledge of the project history and in particular its infrastructure.
+Long-time contributor _@rzats_ visited the project, bringing Ukrainian locale up to date.
+_@qwc_ will be hosting the Valheim server again for our second revival milestone completion celebrations, adding a touch of community spirit as the year slowly comes to its end.
+
+To each contributor, thank you for your dedication, passion, and relentless pursuit of improving Terasology!
+Milestone 2 is not just a checkpoint; it's a testament to what we can achieve together.
+Here's to our community, our contributors, and our potential 🌈
+
+🎉 In total, we merged [27 engine PRs](https://github.com/MovingBlocks/Terasology/pulls?page=1&q=is%3Apr+sort%3Aupdated-desc+is%3Amerged+merged%3A2023-09-11..2023-11-19) and [20 PRs on libraries and side projects](https://github.com/search?q=-repo%3Amovingblocks%2Fterasology+org%3Amovingblocks+is%3Apr+merged%3A2023-09-11..2023-11-19+&type=pullrequests&p=2) such as gestalt, the launcher, or the module site.
+
+## 🚀 Terasology Revival Milestone 3: New Contributors Welcome
+
+Embarking on the journey of our third revival milestone, our repositories are wide open for new contributions by friendly people joining our gooey family! 🌟
+Whether you're a seasoned developer or just getting started, there's a place for you in our dynamic community. Let's build, innovate, and play together! 🚀💬
+
+Let's delve into the tasks that lie ahead, each playing a crucial role in shaping the future of Terasology.
+If you'd like to contribute to a specific task, feel free to ping the respective revivers on our [Discord](https://discord.gg/terasology).
```

Comment:
I'd like to keep it at the top to allow the reader to take not of revivers associated with tasks the reader is interested in on the first read-through to avoid the need for having to go back through the text to find the respective information.
Emphasising optional reading is a good idea, although I'm a bit split on that as I want people to read and follow that remark, so kind of encouraging to not read that part by default... not sure that's what I want 😅 I'll think about that some and then decide what to do with it.

Reply:
I think I'll leave it where it is.